src/components/SieveAccountForm.vue

```diff
@@ -5,107 +5,47 @@
 <template>
 	<form id="sieve-form">
 		<p>
-			<input id="sieve-disabled"
-				v-model="sieveConfig.sieveEnabled"
-				type="radio"
-				class="radio"
-				name="sieve-active"
-				:value="false">
-			<label :class="{primary: !sieveConfig.sieveEnabled}"
-				for="sieve-disabled">
-				{{ t('mail', 'Disabled') }}
-			</label>
-			<input id="sieve-enabled"
-				v-model="sieveConfig.sieveEnabled"
-				type="radio"
-				class="radio"
-				name="sieve-active"
-				:value="true">
-			<label :class="{primary: sieveConfig.sieveEnabled}"
-				for="sieve-enabled">
-				{{ t('mail', 'Enabled') }}
-			</label>
+			<NcCheckboxRadioSwitch :checked.sync="sieveConfig.sieveEnabled">
+				{{ ('mail', 'Enable Sieve Security') }}
+			</NcCheckboxRadioSwitch>
 		</p>
-		<template v-if="sieveConfig.sieveEnabled">
-			<label for="sieve-host">{{ t('mail', 'Sieve Host') }}</label>
-			<input id="sieve-host"
-				v-model="sieveConfig.sieveHost"
-				type="text"
-				required>
+		<div v-if="sieveConfig.sieveEnabled">
+			<NcTextField :label="t('mail', 'Sieve Host')" :value.sync="sieveConfig.sieveHost" />
 			<h4>{{ t('mail', 'Sieve Security') }}</h4>
 			<div class="flex-row">
-				<input id="sieve-sec-none"
-					v-model="sieveConfig.sieveSslMode"
-					type="radio"
-					name="sieve-sec"
-					value="none">
-				<label class="button"
-					for="sieve-sec-none"
-					:class="{primary: sieveConfig.sieveSslMode === 'none'}">{{
-						t('mail', 'None')
-					}}</label>
-				<input id="sieve-sec-ssl"
-					v-model="sieveConfig.sieveSslMode"
-					type="radio"
-					name="sieve-sec"
-					value="ssl">
-				<label class="button"
-					for="sieve-sec-ssl"
-					:class="{primary: sieveConfig.sieveSslMode === 'ssl'}">
+				<ButtonVue :pressed="sieveConfig.sieveSslMode === 'none'"
+					@click="updateSslMode('none')">
+					{{ t('mail', 'None') }}
+				</ButtonVue>
+				<ButtonVue :pressed="sieveConfig.sieveSslMode === 'ssl'"
+					@click="updateSslMode('ssl')">
 					{{ t('mail', 'SSL/TLS') }}
-				</label>
-				<input id="sieve-sec-tls"
-					v-model="sieveConfig.sieveSslMode"
-					type="radio"
-					name="sieve-sec"
-					value="tls">
-				<label class="button"
-					for="sieve-sec-tls"
-					:class="{primary: sieveConfig.sieveSslMode === 'tls'}">
+				</ButtonVue>
+				<ButtonVue :pressed="sieveConfig.sieveSslMode === 'tls'"
+					@click="updateSslMode('tls')">
 					{{ t('mail', 'STARTTLS') }}
-				</label>
+				</ButtonVue>
 			</div>
-			<label for="sieve-port">{{ t('mail', 'Sieve Port') }}</label>
-			<input id="sieve-port"
-				v-model="sieveConfig.sievePort"
-				type="text"
-				required>
+			<NcAppNavigationSpacer />
```

Comment:
```suggestion
```

I think this can't be used outside of app navigation. Looks weird in Firefox because it is rendered as a list element.

![grafik](https://github.com/user-attachments/assets/624fbb66-b85b-404a-92d1-f14c51b1d848)